.github/issue_labeler.yml

```
@@ -1,2 +1,2 @@
 needs_triage:
-  - '.*'
+  - '/.*/'
```

I think this isn't a regex but an fnmatch expression. So if I got it right, this would be something like
```suggestion
  - >-
    **/*
```

---

I wish there was some way to actually test it, but I was following the docs for the action that suggested that regex: https://github.com/marketplace/actions/regex-issue-labeler

---

Ah, I confused it with another action.